Essentials/src/com/earth2me/essentials/Console.java

```diff
@@ -2,15 +2,17 @@
 
 import com.earth2me.essentials.messaging.IMessageRecipient;
 import com.earth2me.essentials.messaging.SimpleMessageRecipient;
-
```

Comment:
Unnecessary diff

Reply:
That's auto formatting again. Usually there's just an empty line between usual imports and static ones. With all due respect, do you have any notes on the actual code?

Reply:
Actual code seems good. (to me atleast)